src/open_inwoner/scss/components/Form/ButtonRadio.scss

```diff
@@ -12,16 +12,38 @@
       color: var(--color-primary);
       cursor: default;
     }
+    &#negative {
```

Comment:
We have nice buttons now!If you think the border on focus is a problem feel free to to fix it. I am not sure about it.

![image](https://user-images.githubusercontent.com/63058071/218735090-0408708b-5fb1-4a7f-ace0-17748125cfab.png)

Reply:
Yes that is not the way it should be, eventually, but I want to collect that in a new ticket for all accessibility issues that are not quite on par yet.